### PR TITLE
provider/kubernetes: Provide deck with details needed for search results

### PR DESCRIPTION
**Pull request by lwander**
@duftler

### CODE REVIEW

**Comment by lwander** [comment]
h/o I'm making one more change to this branch.

**Comment by lwander** [comment]
All set

File: clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiAdaptor.groovy [hunk 1]
```diff
@@ -154,15 +154,17 @@ class KubernetesApiAdaptor {
...
-        cpu: limits?.cpu?.amount,
-        memory: limits?.memory?.amount
-      )
-
-      containerDescription.requests = new KubernetesResourceDescription(
-        cpu: requests?.cpu?.amount,
-        memory: requests?.memory?.amount
-      )
+      containerDescription.limits = limits?.cpu?.amount || limits?.memory?.amount ?
+          new KubernetesResourceDescription(
```
Comment (duftler) [inline clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiAdaptor.groovy]: The indentation of the two blocks does not match,

**Comment by duftler** [comment]
LGTM after addressing comment.

**Pull request merged**